po/es.po

```diff
@@ -0,0 +1,1212 @@
+"files"
+
+#: src\supported-hosts.md:7 src\supported-targets.md:6
+msgid "A combination of CPU architecture, Operating System and ABI"
+msgstr "Una combinación de arquitectura de CPU, Sistema Operativo y ABI"
+
+#: src\supported-hosts.md:9 src\supported-targets.md:8
+msgid "The List"
+msgstr "La Lista"
+
+#: src\supported-hosts.md:11
+msgid "Just `x86_64-unknown-linux-gnu` for now"
+msgstr "Solo `x86_64-unknown-linux-gnu` por ahora"
+
+#: src\supported-hosts.md:13
+msgid "`x86_64-unknown-linux-gnu`"
+msgstr ""
+
+#: src\supported-hosts.md:15
+msgid "We qualified on Ubuntu 18.04 for x86-64"
+msgstr "Calificamos en Ubuntu 18.04 para x86-64"
+
+#: src\supported-hosts.md:16
+msgid "This was driven by the needs of our first client"
+msgstr "Esto fue impulsado por las necesidades de nuestro primer cliente"
+
+#: src\supported-hosts.md:17
+msgid "It comes with GCC 7.5 which is the linker we used for qualification"
+msgstr "Viene con GCC 7.5 el cual es el linker usado en la calificación"
+
+#: src\supported-hosts.md:18
+msgid "More Operating Systems and linkers will be supported in due course"
+msgstr "Tendremos soporte para más Sistemas Operativos y linkers en su momento"
+
+#: src\supported-hosts.md:20
+msgid "But what about $MY_FAVOURITE host"
+msgstr "Pero y qué de $MI_HOST_FAVORITO"
+
+#: src\supported-hosts.md:22 src\supported-targets.md:25
+msgid "Get in touch!"
+msgstr "¡Contáctanos!"
+
+#: src\supported-hosts.md:23
+msgid "If Rust already runs on there, it's a case of porting our test suite"
+msgstr "Si Rust ya corre ahí, requiere portear la suite de tests"
+
+#: src\supported-hosts.md:24
+msgid "If Rust doesn't already run on there ... we can probably fix that for you"
+msgstr "Si Rust todavía no correr ahí ... probablemente lo podemos arreglar para tí"
+
+#: src\supported-targets.md:3
+msgid "Target Platform?"
+msgstr "¿Plataforma Target?"
+
+#: src\supported-targets.md:5
+msgid "The machine you can run the compiled code on"
+msgstr "La máquina que corre el código compilado"
+
+#: src\supported-targets.md:10
+msgid "`x86_64-unknown-linux-gnu` (the supported host)"
+msgstr "`x86_64-unknown-linux-gnu` (el host soportado)"
+
+#: src\supported-targets.md:11 src\supported-targets.md:13
+msgid "`aarch64-unknown-none`"
+msgstr ""
+
+#: src\supported-targets.md:15
+msgid "\"64-bit Arm on bare metal\""
+msgstr "\"64-bit Arm sobre bare metal\""
+
+#: src\supported-targets.md:16
+msgid "Aarch64 mode (A64 instructions) Armv8-A"
+msgstr ""
+
+#: src\supported-targets.md:17
+msgid "`no-std` mode - no Rust `libstd`, only `libcore` and `liballoc`"
+msgstr "modo `no-std` - sin Rust `libstd`, sólo `libcore` y `liballoc`"
+
+#: src\supported-targets.md:18
+msgid "Driven by our lead client's requirements"
+msgstr "Impulsado por los requerimientos de nuestros clientes"
+
+#: src\supported-targets.md:19
+msgid "Bring your existing OS and call its C API from Rust"
+msgstr "Trae tu sistema operative preexistente y llama su API de C desde Rust"
```

Comment:
There is a typo in "operativo"

Reply:
lol, inventé un OS inclusive.
Corregido.